main.cpp

```diff
@@ -113,7 +145,19 @@ Thing 4) school
     3) play sports
  */
 
-
+/*
+Thing 10) audio compressor
+5 properties:
+    1)input
+    2)output
+    3)make up gain
+    4)threshold
+    5)ratio
```

Comment:
Are you sure you can extract 5 properties and 3 actions out of EACH of these?
What are 5 properties of `make-up gain`? 
- [ ] revise.  Consider picking a different Thing for 10)

Reply:
New Thing: Truck?